src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/UserDefinedLanguageClient.java

```diff
@@ -37,4 +42,10 @@ public void dispose() {
         LanguageServerLifecycleManager.getInstance(getProject()).removeLanguageServerLifecycleListener(languageServerStartedListener);
         LanguageServersRegistry.getInstance().removeLanguageServerDefinitionListener(languageServerStartedListener);
     }
+
+    //Hack so that LS requesting workspace/configuration without checking client capability don't cause an error (looking at you rust-analyzer)
+    @Override
+    public CompletableFuture<List<Object>> configuration(ConfigurationParams configurationParams) {
+        return CompletableFuture.completedFuture(Collections.emptyList());

```

Comment:
We need to implement this method by returning proper section by using the configuration settings by loading it in Json and  filter it according section paramètre.

We can do that in 1 separate pr if you wish.